xarray/tests/test_plot.py

```diff
@@ -2997,7 +3001,7 @@ def test_ncaxis_notinstalled_line_plot(self) -> None:
 @requires_matplotlib
 class TestAxesKwargs:
     @pytest.fixture(params=[1, 2, 3])
-    def data_array(self, request):
+    def data_array(self, request) -> DataArray:  # type: ignore[return] # error: Missing return statement  [return]
```

Comment:
I don't understand why it complains about missing return type here. Maybe there's something strange with the `pytest.fixture`.

Reply:
I think it's because if `dims>3` then nothing is returned. So we could put a `ValueError` at the bottom

Reply:
Works! Nice catch @max-sixty !